### PR TITLE
Hunger story

### PR DESCRIPTION
**Pull request by jitendar7**
Please review this, @casey-collab

### CODE REVIEW

File: english/hungry/hungry.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+When ur hungry, wat do you eat ?
```
Comment (casey-collab) [inline english/hungry/hungry.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by jitendar7** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :smiley_cat:

**Pull request merged**